### DISCUSSION TITLE
Firmware Jumper t-lite v2

### DISCUSSION THREAD

**Discussion by Shlipez** [EdgeTX/edgetx#5697] [Q&A] [answered]
Please help me find where I can flash my controller, I have searched all the sites, but there is only firmware on jumper t-lite

**Comment by pfeerick** [comment, marked as answer]
The T-Lite V2 doesn't need it's own firmware variant - i.e. use the T-Lite firmware on it. 

If you are ever unsure as to what firmware variant you need for any handset, check what it is currently using on the Radio Settings / SYS -> Version page, as it tells you at the end of the firmware line ;)

![image](https://github.com/user-attachments/assets/7cc8d23e-5542-4a61-ac27-cdf6e8212636)
